ugsync/src/main/java/org/apache/ranger/unixusersync/process/PolicyMgrUserGroupBuilder.java

```diff
@@ -419,6 +420,14 @@ public void addOrUpdateUser(String userName, List<String> groups) throws Throwab
 					}
 				}
 			}
+
+			// Update user role depending on the group changes
+			boolean isUserCurrentlyPortalAdmin = isUserInAdminGroup(groups);
```

Comment:
Do you use these booleans anywhere else? It might be even more readable to simply use methods in the if statement. But, it is really your preference. Clearly, it's a personal preference how you write these.

Reply:
Nope. This is the only place I use them. I was also not sure while coding which one makes more sense. I wanted to write in this way to make it easier for someone reading the code (it  might not be very necessary, though). let me keep it in this way for now.